pkg/webhook/resources/virtualmachine/mutator_test.go

```diff
@@ -169,7 +196,7 @@ func TestPatchAffinity(t *testing.T) {
 									{
 										MatchExpressions: []v1.NodeSelectorRequirement{
 											{
-												Key:      "network.harvesterhci.io/test",
+												Key:      "network.harvesterhci.io/bbbb",
```

Comment:
prefer to replace `bbbb` with `to-delete-after-mutating` as it is more readable.